### PR TITLE
[IMP] accounting: reconciliation models

### PR DESCRIPTION
**Pull request by toaa-odoo**
Updated reconciliation page. Forward to V15.

taskid: 3043106

### CODE REVIEW

**Review by chiaraprattico — APPROVED** [review]

File: content/applications/finance/accounting/bank/reconciliation/reconciliation_models.rst [hunk 1]
```diff
@@ -31,96 +28,96 @@ There are three types of Reconciliation Models:
...
+   not to delete it.
 
-Open the model you want to modify, or click on *Create* to create a new one, then fill out the form.
+Open the model you want to modify or click on :guilabel:`Create` to create a new one, and fill out
+the form.
 
 Type
 ----
 
 See :ref:`above <reconciliation_models_types>` for an explanation about the different types of
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/bank/reconciliation/reconciliation_models.rst:88]:
```suggestion
See :ref:`types of reconciliation models section <reconciliation_models_types>` for an explanation about the different types of
```

File: content/applications/finance/accounting/bank/reconciliation/reconciliation_models.rst [hunk 2]
```diff
@@ -31,96 +28,96 @@ There are three types of Reconciliation Models:
...
+reconciliation models.
 
 .. note::
-   If the *Documents* application is installed on your database, an additional **Activity type**
-   field appears when *To check* is ticked. Selecting the value *Reconciliation request* implies
-   that, whenever you use this model, a *Request Document* window pops up to request a document from
-   a user.
+   If the *Documents* application is installed on your database, an additional
+   :guilabel:`Activity type` field appears when :guilabel:`To Check` is ticked. Selecting the value
+   `Reconciliation request` implies that whenever you use this model, a :guilabel:`Request Document`
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/bank/reconciliation/reconciliation_models.rst:94]:
```suggestion
   :guilabel:`Reconciliation request` implies that whenever you use this model, a :guilabel:`Request Document`
```
I think we still need to use guilabels for field values

File: content/applications/finance/accounting/bank/reconciliation/reconciliation_models.rst [hunk 3]
```diff
@@ -31,96 +28,96 @@ There are three types of Reconciliation Models:
...
+   window pops up to request a document from the user.
 
-Conditions on Bank Statement Line
+Conditions on bank statement line
 ---------------------------------
 
-Define here all the conditions that are required for a Reconciliation Model to be applied.
+Define here all the conditions you wish to apply for a reconciliation model to be applied. Depending
+on the :guilabel:`Type` of models you chose, different options are available.
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/bank/reconciliation/reconciliation_models.rst:101]:
```suggestion
Define here all the conditions that should be fulfilled for the reconciliation model to be applied. Depending
on the :guilabel:`Type` of model you choose, different options are available.
```

> to apply for a reconciliation model to be applied

I would rephrase that. Feel free to use something better than my suggestion though ;)

File: content/applications/finance/accounting/bank/reconciliation/reconciliation_models.rst [hunk 4]
```diff
@@ -31,96 +28,96 @@ There are three types of Reconciliation Models:
...
-Counterpart Values
+Counterpart values
 ------------------
 
-This section comprises the values that are applied by the Reconciliation Model.
-
-If the value to reconcile needs to be written-off in two separate accounts, click on *Add a second
-line*.
+This section comprises the values that are applied by the reconciliation model. If the value to
+reconcile needs to be written-off in two separate accounts, click on :guilabel:`Add a second line`.
```
Comment (xpl-odoo) [inline content/applications/finance/accounting/bank/reconciliation/reconciliation_models.rst:116]:
```suggestion
reconcile needs to be written-off in two separate accounts, click on :guilabel:`Add a line` a second time.
```

**Review by xpl-odoo — CHANGES_REQUESTED** [review]
Nice job Tom!🔥

**Review by xpl-odoo — CHANGES_REQUESTED** [review]
Regarding image file names (reconciliation_models_xxx_yyy.png)

- remove _reconciliation_models_ from all files
- replace underscores with hyphens

-> xxx-yyy.png

File: content/applications/finance/accounting/bank/reconciliation/reconciliation_models.rst [hunk 5]
```diff
@@ -31,96 +28,96 @@ There are three types of Reconciliation Models:
...
 
 Configuration
 =============
 
-To manage or create new **Reconciliation Models**, go to :menuselection:`Accounting -->
-Reconciliation --> Reconciliation Models`. Alternatively, you can also open this menu from the
-Accounting Overview, by going to your Bank Journal card, clicking on the three little dots, and then
-on *Reconciliation Models*.
+To manage or create new **reconciliation models**, go to :menuselection:`Accounting dashboard -->
+Configuration --> Reconciliation Models`. Alternatively, you can also open this menu from
```
[the remark this answers is not in the document]
Comment (toaa-odoo) [reply content/applications/finance/accounting/bank/reconciliation/reconciliation_models.rst:70]: True, it's because "Accounting" is also a menu within the accounting app and therefore if a user were to go to accounting and then configuration, he wouldn't find it, because configuration is a menu itself not part of the "accounting" menu.

**Comment by toaa-odoo** [comment]
> Nice job Tom!fire

Thanks for your review! I accepted everything :)

**Review by xpl-odoo — APPROVED** [review]
They should rename the Accounting menu 😵
In the meantime, mentioning "Accounting app" and "Accounting menu" is the most elegant solution I can think of

**Comment by xpl-odoo** [comment]
> Updated reconciliation page. Forward to V15.
> 
> taskid: 4720

Btw, you copied the `menu_id` instead of the `id` (3043106) in your commit message :nitpick:

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Comment by AntoineVDV** [comment]
@robodoo r-

**Comment by jcs-odoo** [comment]
@fw-bot up to 15.0
